config/main.py

```diff
@@ -6485,6 +6485,42 @@ def polling_int(ctx, interval):
     except ValueError as e:
         ctx.fail("Invalid ConfigDB. Error: {}".format(e))
 
+def is_port_egress_sflow_supported():
+    state_db = SonicV2Connector(host='127.0.0.1')
+    state_db.connect(state_db.STATE_DB, False)
+    entry_name="SWITCH_CAPABILITY|switch"
+    supported = state_db.get(state_db.STATE_DB, entry_name,"PORT_EGRESS_SAMPLE_CAPABLE")
+    click.echo("switch_capability-egress-sflow {}".format(supported))
+    return supported
+
+#
+# 'sflow' command ('config sflow sample-direction ...')
+#
+@sflow.command('sample-direction')
+@click.argument('direction',  metavar='<sample_direction>', required=True, type=str)
+@click.pass_context
+def sample_dir(ctx, direction):
```

Comment:
Please consider using a more readable function name, because it intersects with another function called sample_direction()

sample_dir() --> global_sample_direction()

Reply:
> Please consider using a more readable function name, because it intersects with another function called sample_direction()
> 
> sample_dir() --> global_sample_direction()

Will rename to  global_sample_direction()

Reply:
Will rename to global_sample_direction()